sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -1144,11 +1144,9 @@ export class KeyClient {
       keyType: keyBundle.key ? keyBundle.key.kty : undefined,
       properties: {
         id: keyBundle.key ? keyBundle.key.kid : undefined,
-        name: parsedId.name,
```

Comment:
Ooo, so did TypeScript 3.9 actually warn about this now? If so that's really cool.

Reply:
yeah it complains about the property being overridden by the splat